src/app/components/flights-list/flights-list.test.tsx

```diff
@@ -0,0 +1,249 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { FlightSearchInfo, FlightsStateProps, SortBy, Stops } from "../../models/flight-parameters";
+import FlightsList from "./flights-list";
+import { FlightsContext } from "../../contexts/flights-context";
+import * as Filters from "../filters/filters";
+import { FlightSortBy, FlightSortByProps } from "../sort-by/sort-by";
+
+jest.mock('../filters/filters', () => ({
+  __esModule: true,
+  default: ({ ...props }: any) => <div {...props}></div>
+}));
+
+jest.mock('./flight-row/flight-row', () => ({
+  __esModule: true,
+  FlightRow: (flight: FlightSearchInfo) => <div className="row">{flight.airline_logo}</div>
+}));
+
+jest.mock('../../../components/ui/skeleton', () => ({
+  Skeleton: ({...props }) => <div {...props} >Loading...</div>
+}));
+
+var mockedSortBy = jest.fn();
+jest.mock('../sort-by/sort-by', () => ({
+  FlightSortBy: ({...props}) => <div {...props}></div>
+}));
+
+describe(FlightsList.name, () => {
+  let contextMock: FlightsStateProps;
+  beforeEach(() => {
+    contextMock = {
+      flights: [],
+      isLoading: false,
+      airports: [],
+      parameters: {},
+      fetchFlights: jest.fn(),
+      setFlightParameters: jest.fn()
+    };
+
+    jest.clearAllMocks();
+  });
+  describe('Sorting', () => {
+    let contextValue: FlightsStateProps, container: HTMLElement;
+    beforeEach(() => {
+      contextValue = {
+        ...contextMock,
+        flights: [
```

Comment:
use a method to get this value so the test is readable

Reply:
I was wanting to discuss about the tests for this file. 
Would we prefer to test the helper functions that are used to sort and filter the flights separately and for the component tests, mock the helper function and test if they are being called under different circumstances